### ISSUE TITLE
python jediEnabled

### ISSUE THREAD

**Issue by lvytwjw** [MicrosoftDocs/intellicode#69]
**Product and Version** [VSCode]:
**OS Version** [Windows]:
**IntelliCode Extension Version**: 1.14
**Target Platform or Language** [python]:

**Steps to Reproduce / Scenario:**

1.when i us Visual Studio IntelliCode, '"python.jediEnabled": true' in setting.json
2.then i can't get some IntelliCode. like define a  variate as str :a='' then str function can't show in the list

**Comment by jkeech** [comment]
This is by design. IntelliCode Python support requires the Microsoft Python Language Server in order for the language server plugin to work. This means that it only works with `"python.jediEnabled": false`. The IntelliCode extension is supposed to prompt you with a toast in this case:

![image](https://user-images.githubusercontent.com/746020/54046398-04e5ba00-4189-11e9-9d46-7325be5915c5.png)

**Comment by markw-t** [comment]
@lvytwjw , do you see the notification that @jkeech mentions above when you try to use IntelliCode with jediEnabled true?

**Comment by RicaBenhossi** [comment]
I was facing the same issue. I don't know if it was a solution but in order to test, I open a new VSCode window, without a workspace or anything else, just a blank plain text file. I changed it to python language (using command Change Language Mode). Didn't work. But, when I opened my project again, it suddenly back working. 

As I said, I don't know if it was a solution, but you can try these steps:
1. Open a new VSCode window, without a workspace, project, etc.
2. Open a new file.
3. Change the Language Mode of the file to Python (Ctrl+Shift+P then type Change Language Mode)
4. Try typing some command.
5. If it doesn't work, close VSCode and open it again.
6. Open your workspace and test it again.

**Comment by jkeech** [comment]
@RicaBenhossi, which version of the Python language server do you have (you can check from the Python output window)? I have seen that before with the older versions of the language server, but the newer versions seem to work better with files that are not yet saved to disk.

**Comment by RicaBenhossi** [comment]
I've installed Microsoft Python Language Server version 0.2.37.0 and Python 3.7.1. 

VSCode
Version: 1.33.0-insider
Commit: 4039ea871879ef8f654f27df5c32d209104b3a44
Date: 2019-03-29T10:48:26.004Z
Electron: 3.1.6
Chrome: 66.0.3359.181
Node.js: 10.2.0
V8: 6.6.346.32
OS: Linux x64 4.15.0-46-generic

**Comment by RicaBenhossi** [comment]
I don't know if it is a new issue or is related with this one, but I noticed today that, although it might be working for me, when I choose a function like list(), dict(), print(), input() it doesn't add the parenthesis. I'm pretty sure that it brings us the parenthesis before...

Is this a new issue? Should I open a new one if it is or we keep it here?

**Comment by jkeech** [comment]
@ricabenhossi, can you open a new issue for that? It will help if you also temporarily disable the IntelliCode extension and reload the VS Code window to confirm if the behavior around parenthesis changes. Thanks!

**Comment by RicaBenhossi** [comment]
That's odd... Even if I disable Intellicode extension, VSCode keeps suggesting me commands without adding parenthesis though. I don't know it is expected, but I've made some tests:
Perform a fresh install of VSCode and extensions -> no results
Install an old version (from more than one month ago) of IntelliCode extension and Python extension (ms-python.python) -> no results
Keep only the extensions above for python and nothing more -> no results either

As you requested @jkeech, I've opened a new issue [Not adding parenthesis after python #80](https://github.com/MicrosoftDocs/intellicode/issues/80). Can it be related with [autoComplete addBrackets was doesn't work #10](https://github.com/MicrosoftDocs/intellicode/issues/10)?
